.github/workflows/test.yml

```diff
@@ -32,3 +32,30 @@ jobs:
           go get -v -t -d ./...
       - name: Test
         run: go test ./...
+
+  android-client:
+    name: Android Client Test
+    runs-on: ubuntu-latest
+    services:
+      threads:
+        image: textile/go-threads
+        ports:
+        - 6006:6006
+    steps:
+      - name: set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Install protoc
+        uses: Arduino/actions/setup-protoc@master
+        with:
+          version: '3.11.2'
+      - name: Install Android SDK
+        uses: malinskiy/action-android/install-sdk@release/0.0.5
+      - name: Check out code
+        uses: actions/checkout@v1
+      - name: Test
+        run: |
+             cd api/pb/android
+             ./gradlew threads:test
```

Comment:
these run the unit tests in https://github.com/textileio/go-threads/pull/231/files#diff-4804adf0dc5fc30a630683638d98519e